next.config.js

```diff
@@ -0,0 +1,8 @@
+const withPWA = require('next-pwa')
+
+module.exports = withPWA({
+  pwa: {
+    dest: 'public',
```

Comment:
A quick suggestion taken from my website:

https://github.com/carloscuesta/carloscuesta.me/blob/182df959916ce586437b336d00651738524ce1d4/next.config.js#L5

```suggestion
    disable: process.env.NODE_ENV === 'development',
    dest: 'public',
```